util/pip_package/open3d/linux/readme.txt

```diff
@@ -0,0 +1 @@
+place linux binaries here
```

Comment:
Question: is this just place holder?

Reply:
it's a reminder for whomever is building the pip package.